custom/icds_reports/reports/new_born_with_low_weight.py

```diff
@@ -27,7 +27,7 @@ def get_data_for(filters):
             '%s_name' % loc_level, '%s_map_location_name' % loc_level
         ).annotate(
             low_birth=Sum('low_birth_weight_in_month'),
-            in_month=Sum('born_in_month'),
+            in_month=Sum('weighed_and_born_in_month'),
```

Comment:
@lwyszomi You'll still need born_in_month to calculate % unweighed and number of newborns born in a month. I believe that